torch/csrc/jit/pickler.cpp

```diff
@@ -723,6 +729,48 @@ OpCode Unpickler::readInstruction() {
               AT_ERROR("Unknown pickler class id");
           }
         });
+      } else if (
+          module_name == "torch._utils" && class_name == "_rebuild_tensor_v2") {
```

Comment:
would be nice to extract to a subfunction and explicitly document in both python and C++ that those functions should match semantically